js_modules/dagster-ui/packages/ui-core/src/instance/InstanceConcurrency.tsx

```diff
@@ -112,6 +135,79 @@ type DialogAction =
     }
   | undefined;
 
+const RunConcurrencyContent = ({
+  hasRunQueue,
+  runQueueConfig,
+}: {
+  hasRunQueue: boolean;
+  runQueueConfig: any;
+}) => {
+  if (!hasRunQueue) {
+    return (
+      <>
+        <Box padding={{vertical: 16, horizontal: 24}} border="bottom">
+          <Subheading>Run concurrency</Subheading>
+        </Box>
+        <Box>
```

Comment:
Nbd, but if you don't have any Box props to apply, just a vanilla  `div` is fine